### ISSUE TITLE
Global installation not adding the path

### ISSUE THREAD

**Issue by angel-luis** [oven-sh/bun#7136]
### What version of Bun is running?

1.0.11

### What platform is your computer?

Darwin 23.0.0 arm64 arm

### What steps can reproduce the bug?

`bun install -g eas-cli`

### What is the expected behavior?

To use `eas` command.

### What do you see instead?

`zsh: command not found: eas`

### Additional information

Working well with: `npm install -g eas-cli`

**Comment by back-2-95** [comment]
I just got this when I was building Bun docker image and installed a cli app with `un install --global myapp` and command is not found.

If I knew where it's installed I could add it to path.

**Comment by Sophist-UK** [comment]
The issue here is (IMO) that the `bun` installer adds some stuff to `~/.bashrc` to add `bun` to the path, but it does not look to see if it needs to do something similar to `~/.zshrc`.

**Comment by emekaorji** [comment]
Having this same issue, anything here? Still works with `npm i -g [package]` though.

**Comment by Sophist-UK** [comment]
Yes - in the end I added a global install to my docker build (which is what I really needed anyway).

**Comment by alex20465** [comment]
Same issue, global installed packages are not available to execute.

I would extend the PATH var myself, but where does the bun docker install the global packages ?

**Comment by DrSensor** [comment]
npm has this to inspect the global install path
```sh
npm ls -g
```
does bun has something similar?

**Comment by SC0d3r** [comment]
I have this issue and I have done these steps still running `RUN which <pkg>` returns non zero which means cannot find the installed pkg
```js
RUN bun install -g <pkg>
ENV PATH="/home/path/to/.bun/bin:${PATH}"
ENV PATH="${PATH}:${BUN_INSTALL_BIN}" 
RUN which <pkg> // gives non zero exit which means doesnt find it
```
the strange thing is that if I keep the container running and then exec into that container and run which <pkg> it finds the pkg

**Comment by TomasHubelbauer** [comment]
To anyone who might run into a similar problem as me: make sure the actual problem is the binary not being in PATH and not the binary running and failing to find Node. I skimmed the error message and figured out Bun was the issue, but I didn't realize I was on a system with only Bun, not Node and that the global binary I installed had a shebang for Node, so when it started, it failed right away trying to find Node. I patched the shebang of the global binary to use Bun, not Node, and I was off on my merry way.
